ss-local: add page (#4857)
tldr-pages/tldr: pages/common/ss-local.md

# ss-local

> > Shadowsocks client as SOCKS5 proxy.

Reviewer: Perhaps this will read better?
```suggestion
> Run a Shadowsocks client as a SOCKS5 proxy.
```

> More information: <https://github.com/shadowsocks/shadowsocks-libev/blob/master/doc/ss-local.asciidoc>.

> - Run Shadowsocks proxy by specifying host, server port, local port, password, and encrypt method:

Reviewer: Perhaps this will read better?
```suggestion
- Run a Shadowsocks proxy by specifying the host, server port, local port, password, and encryption method:
```

`ss-local -s {{host}} -p {{server_port}} -l {{local port}} -k {{password}} -m {{encrypt_method}}`

> - Run Shadowsocks proxy by specifying the config file:

Reviewer: Perhaps this will read better?
```suggestion
- Run a Shadowsocks proxy by specifying the config file:
```

`ss-local -c {{path/to/config/file.json}}`

> - Use plugin to run the proxy client:

Reviewer: Perhaps this will read better?
```suggestion
- Use a plugin to run the proxy client:
```
